objectscale-manager/templates/objectscale-manager-app-configmap.yaml

```
@@ -27,6 +27,16 @@ data:
           args:
             - -target-version
             - {{ default .Values.tag .Values.healthChecks.preUpdate.image.tag}}
+        - name: cloudiq-inventory-report
+          container: {{.Values.global.registry}}/objectstore-inventory:{{.Values.tag}}
+          serviceaccount: {{ .Release.Name }}-healthchecks
+          schedule: "0 */6 * * *"
+          timelimit: "5m"
+        - name: cloudiq-health-report
```

remove report so that we can expand this if needed.

```suggestion
        - name: cloudiq-health
```

---

Fixed